guacamole-common/src/main/java/org/apache/guacamole/protocol/ConfiguredGuacamoleSocket.java

```diff
@@ -142,6 +150,12 @@ public ConfiguredGuacamoleSocket(GuacamoleSocket socket,
 
             // Retrieve argument name
             String arg_name = arg_names.get(i);
+            
+            // Check for protocol version as first argument
+            if (i == 0 && arg_name.startsWith("VERSION_")) {
+                protocol = GuacamoleProtocolVersion.valueOf(arg_name);
+                arg_values[i] = protocol.toString();
```

Comment:
This will be overwritten by:

https://github.com/apache/guacamole-client/blob/0f4ff0fe468e064a672296af47c0744cf919ee4e/guacamole-common/src/main/java/org/apache/guacamole/protocol/ConfiguredGuacamoleSocket.java#L160-L167

Reply:
Yeah, that could be a problem...should be fixed.